presto-raptor/src/main/java/com/facebook/presto/raptor/util/UuidUtil.java

```diff
@@ -126,4 +128,15 @@ public static UUID uuidFromBytes(byte[] bytes)
                 .putLong(uuid.getLeastSignificantBits())
                 .array();
     }
+
+    /**
+     * @param uuidSlice String representation of UUID
+     * @return Slice over byte representation of UUID
+     */
+    public static Slice uuidStringToBytes(Slice uuidSlice)
+    {
+        UUID uuid = UUID.fromString(new String(uuidSlice.getBytes()));
```

Comment:
`uuidSlice.toStringUtf8()`

The `String(byte[])` constructor should never be used because it decodes using the JVM default charset